src/net/dhcp.rs

```diff
@@ -249,6 +252,26 @@ pub fn set_pxe_reply(pxe_reply_packet: &Dhcpv4Packet) -> Result<()> {
     Ok(())
 }
 
+pub fn get_file_size(server_ip: &IpAddr, filename: &NullTerminatedAsciiStr) -> Result<u64> {
```

Comment:
Should we rename this to `mtftp_get_file_size`?

Reply:
Done